packages/shell-api/src/field-level-encryption.spec.ts

```diff
@@ -962,7 +962,7 @@ srDVjIT3LsvTqw==`,
               return;
             }
             expect.fail('missed exception');
-            break;
+          // eslint-disable-next-line no-fallthrough
```

Comment:
might be coming from a merge conflict or my own changes but is this an intentional choice? `.fail` will essentially throw so no use in the break but break seems preferable to a eslint override